.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,119 +1,146 @@
-name: Bug Report
-description: Helps us improve our product!
-labels: ["Needs triage", "[Type] Bug"]
-type: 'Bug'
+name: IPSM Bug Report v2
+description: Help us improve our product!
+labels: "Needs triage"
```

I would keep the Bug type label as well:

```suggestion
labels: ["Needs triage", "[Type] Bug"]
```

---

Done in 5f4385b00bfe5b6c0cb6c2a1c33615727152ae9a